spa/src/utils/dynamicTableHelper.tsx

```diff
@@ -106,16 +115,25 @@ export const getGHSubscriptionsRows = (
 					content: (
 						<div css={rowWrapperStyle}>
 							<span>
-								{cloudConnection.isGlobalInstall
-									? `All repos`
-									: `Only select repos`}
+								<a href={cloudConnection.html_url} target="_blank">
```

Comment:
This URL would be incorrect. The repos should redirect to `https://XXXXXX.atlassian.net/plugins/servlet/ac/com.github.integration.XXXX/gh-addon-subscription-repos?ac.subscriptionId=XXX`
This would be a connect page redirection, so please use `AP.navigator.go` instead of `<a>` tags. ([for more details](https://developer.atlassian.com/cloud/jira/platform/jsapi/navigator/)). Please see how its being used in `jira-configuration.js#340` for reference.